demo/Cimpress.Nancy.Demo/DemoLogger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cimpress.Nancy.Components;
+
+namespace Cimpress.Nancy.Demo
+{
+    public class DemoLogger : INancyLogger
+    {
+        public void Configure(IDictionary<string, string> options)
+        {
+
+        }
+
+        public void Debug(object data)
+        {
+            Console.WriteLine($"Debug: {data}");

```

Comment:
When I ran the demo, I couldn't see anything being printed to the output window in VS. Where should it go?